ewc-rfc001-issue-verifiable-credential.md

```diff
@@ -60,6 +60,12 @@ The credential offer can be an authorisation flow or a pre-authorised one, as sh
 ![Alt text](images/issuer-authorisation-code-flow.png)
 Figure 1: Issuance using Authorisation Code Flow based on [1]
 
+The figure does not seem to implement the authorization code flow, 
```

Comment:
Agree. I have raised an issue for the same (https://github.com/EWC-consortium/eudi-wallet-rfcs/issues/6)